static/app/views/replays/replayTable.tsx

```diff
@@ -72,46 +74,78 @@ function SortableHeader({
   );
 }
 
-function ReplayTable({isFetching, replays, showProjectColumn, sort}: Props) {
+function ReplayTable({isFetching, replays, showProjectColumn, sort, fetchError}: Props) {
   const organization = useOrganization();
   const theme = useTheme();
   const minWidthIsSmall = useMedia(`(min-width: ${theme.breakpoints.small})`);
 
+  const tableHeaders = [
+    t('Session'),
+    showProjectColumn && minWidthIsSmall ? (
+      <SortableHeader
+        key="projectId"
+        sort={sort}
+        fieldName="projectId"
+        label={t('Project')}
+      />
+    ) : null,
+    <SortableHeader
+      key="startedAt"
+      sort={sort}
+      fieldName="startedAt"
+      label={t('Start Time')}
+    />,
+    <SortableHeader
+      key="duration"
+      sort={sort}
+      fieldName="duration"
+      label={t('Duration')}
+    />,
+    <SortableHeader
+      key="countErrors"
+      sort={sort}
+      fieldName="countErrors"
+      label={t('Errors')}
+    />,
+    t('Activity'),
+  ].filter(Boolean);
+
+  if (fetchError && !isFetching) {
+    const reasons = [
+      t('The search parameters you selected are invalid in some way'),
+      t('There is an internal systems error or active issue'),
+    ];
+
+    return (
+      <StyledPanelTable
+        headers={tableHeaders}
+        showProjectColumn={showProjectColumn}
+        isLoading={false}
+      >
+        <StyledDetailedError
+          hideSupportLinks
+          heading={t('Sorry, the list of replays could not be found.')}
+          message={
+            <div>
+              <p>{t('This could be due to a handful of reasons:')}</p>
+              <ol className="detailed-error-list">
+                {reasons.map((reason, i) => (
+                  <li key={i}>{reason}</li>
+                ))}
+              </ol>
+            </div>
+          }
+        />
```

Comment:
Let's use `<Alert>` - https://storybook.sentry.dev/?path=/story/components-alerts-alert--with-icons

Example on issue stream:
![image](https://user-images.githubusercontent.com/79684/188682391-80a0b7b9-7956-4f97-81bc-def6b122e6c3.png)


(although there shouldn't be a margin-bottom there)